wings/watcher/weth_watcher.py

```diff
@@ -58,19 +57,19 @@ def __init__(self,
         self._contract_event_logger = ContractEventLogger(w3, weth_token.address, weth_token.abi)
         self._poll_weth_logs_task: asyncio.Task = None
         self._event_forwarder: EventForwarder = EventForwarder(self.did_receive_new_blocks)
-        self._new_blocks_queue: asyncio.Queue = None
+        self._new_blocks_queue: asyncio.Queue = asyncio.Queue()
 
-    def start(self):
-        self._new_blocks_queue = asyncio.Queue()
+    async def start_network(self):
+        if self._poll_weth_logs_task is not None:
+            await self.stop_network()
         self._blocks_watcher.add_listener(NewBlocksWatcherEvent.NewBlocks, self._event_forwarder)
         self._poll_weth_logs_task = asyncio.ensure_future(self.poll_weth_logs_loop())
 
-    def stop(self):
-        self._blocks_watcher.remove_listener(NewBlocksWatcherEvent.NewBlocks, self._event_forwarder)
+    async def stop_network(self):
         if self._poll_weth_logs_task is not None:
             self._poll_weth_logs_task.cancel()
             self._poll_weth_logs_task = None
-        self._new_blocks_queue = None
```

Comment:
why was this line removed?

Reply:
Same reason as the ERC20 events watcher - we want the WETH watcher to resume getting events from the oldest known non-fetched block when the network comes back online.